dali/operators/generic/slice/slice_attr.h

```diff
@@ -103,12 +103,22 @@ class SliceAttr {
         for (size_t i = 0; i < axes.size(); i++) {
           auto dim = axes[i];
           float anchor_val = slice_anchor_data[i];
-          if (normalized_anchor_)
-            anchor_val *= shape[dim];
           float shape_val = slice_shape_data[i];
-          if (normalized_shape_)
+          int64_t slice_end;
+          // special case - minimize the floating point error by multiplying only once after sum
+          if (normalized_anchor_ && normalized_shape_) {
+            slice_end = static_cast<int64_t>((anchor_val + shape_val) * shape[dim]);
```

Comment:
In my opinion, we should be rounding, not truncating.

Reply:
...and the loss of precision, if any, is rather marginal. If we care for extreme cases (slicing huge 1D objects, for example), this should be a double anyway.

Reply:
It is marginal but still there is a divergence between implementations. So I wanted to make that consistent. Will switch to round.

Reply:
Done